bindings/python/README.md

```diff
@@ -0,0 +1,48 @@
+# libiio: Python Bindings
+
+This package contains the python bindings for libiio, a library for interfacing with Linux IIO devices.
+
+libiio is used to interface to the Linux Industrial Input/Output (IIO) Subsystem. The Linux IIO subsystem is intended to provide support for devices that in some sense are analog to digital or digital to analog converters (ADCs, DACs). This includes, but is not limited to ADCs, Accelerometers, Gyros, IMUs, Capacitance to Digital Converters (CDCs), Pressure Sensors, Color, Light and Proximity Sensors, Temperature Sensors, Magnetometers, DACs, DDS (Direct Digital Synthesis), PLLs (Phase Locked Loops), Variable/Programmable Gain Amplifiers (VGA, PGA), and RF transceivers. You can use libiio natively on an embedded Linux target (local mode), or use libiio to communicate remotely to that same target from a host Linux, Windows or MAC over USB or Ethernet or Serial.
+
+[![Build Status](https://travis-ci.org/analogdevicesinc/libiio.svg?branch=master)](https://travis-ci.org/analogdevicesinc/libiio)
+[![PyPI version](https://badge.fury.io/py/libiio.svg)](https://badge.fury.io/py/libiio) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/4bd027bfc5774029a30a9e1cedf5a434)](https://www.codacy.com/app/rgetz/libiio?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=analogdevicesinc/libiio&amp;utm_campaign=Badge_Grade)
+![open bugs](https://img.shields.io/github/issues/analogdevicesinc/libiio.svg)
+
+[[Docs](https://analogdevicesinc.github.io/libiio/v0.19/python/index.html)]
+[[Support](http://ez.analog.com)]
+[[Wiki](https://wiki.analog.com/resources/tools-software/linux-software/libiio)]
+
+### Requirements
+To use these bindings naturally you need the core library they depend upon, libiio. This is not packaged with the pypi release but there are a number of options:
+  - If you want to just use libiio, we suggest using the [latest release](https://github.com/analogdevicesinc/libiio/releases/latest).
+  - If you think you have found a bug in the release, or need a feature which isn't in the release, try the latest **untested** binaries from the master branch and check out the [documentation](https://codedocs.xyz/analogdevicesinc/libiio/) based on the master branch. We provide builds for a few operating systems. If you need something else, we can most likely add that -- just ask.
+
+### Installing the bindings
+To install these bindings there are a few methods. If you already have the library itself and just need the bindings, pip is the most convenient method:
+```
+(sudo) pip install libiio
+```
+If you do not want to use pip, then installation is dependent on your operating system.
+#### Linux / macOS
+For Linux and macOS the python bindings need to be installed through source if not using pip. For v0.20 and beyond this requires the `-DPYTHON_BINDINGS=ON` flag during the cmake configuration. Further documentation is located [here](https://github.com/analogdevicesinc/libiio/blob/master/README_BUILD.md).
```

Comment:
It wasn't 0.20, it was 0.19

The change was made Fri Jan 17 (f1faa2816c076eaea285ac4683fefa1c4f5878c9) 
0.19 was released Feb 14 (5f5af2e417129ad8f4e05fc5c1b730f0694dca12)

Reply:
Solved.